segment_sc_contrast-agnostic/run_inference_single_image.py

```diff
@@ -240,6 +240,36 @@ def keep_largest_object(predictions):
     return predictions
 
 
+# Adapted from ivadomed:
+# https://github.com/ivadomed/ivadomed/blob/e101ebea632683d67deab3c50dd6b372207de2a9/ivadomed/postprocessing.py#L224-L245
+def remove_small_objects(data, size_min):
```

Comment:
i suggest working with atomic PRs-- ie: first, merge this one, and then create a new PR where you implement the remove_small functionality